index.html

```diff
@@ -0,0 +1,131 @@
+      across. </h3>
+    </div>
+
+  </section>
+
+
+  <!--News-->
+  <h1 class="section">NEWS</h1>
+  <section class="news">
+
+    <div class="news-box-pineapple">
+      <a class="news-box-image"> <img
+          src="https://ichef.bbci.co.uk/news/1536/cpsprodpb/b331/live/0db6aee0-69e8-11ef-8ae1-911441021d16.jpg.webp"
+          alt="pineapples"></a>
+      <a class="news-box-headline" href="https://www.bbc.com/news/articles/c8rx2xvj237o">Spain’s 'pineapple-gate'
+        sparks hopes of romance and shop chaos</a>
+    </div>
+
+    <div class="news-box-mango">
+      <a class="news-box-image"> <img src="https://cdn.freshfruitportal.com/2022/12/mango-Keitt-2-1024x681-1.png"
+          alt="mango"></a>
+      <a class="news-box-headline"
+        href="https://www.freshfruitportal.com/news/2024/10/07/mango-shipments-from-october-to-november-expected-to-increase/">Mango
+        shipments to the US from October to November expected to increase</a>
+    </div>
+
+    <div class="news-box-passionfruit">
+      <a class="news-box-image"> <img
+          src="https://www.ft.com/__origami/service/image/v2/images/raw/ftcms%3A91fc67dd-d0d3-4c83-831c-6421a0b181f6?source=next-article&fit=scale-down&quality=highest&width=700&dpr=2"
+          alt="mango"></a>
+      <a class="news-box-headline" href="https://www.ft.com/content/6d261198-807b-41c0-85d4-bd7c09e9f2fc">The
+        watchmaking odd couple and their passion (fruit) project</a>
+    </div>
+
+    <div class="news-box-pomelo">
+      <a class="news-box-image"> <img
+          src="https://media.nationthailand.com/uploads/images/md/2022/02/1lOkijL3j5FXBm0wa3B6.webp" alt="mango"></a>
+      <a class="news-box-headline" href="https://www.nationthailand.com/business/40011862">US greenlights import of
+        pomelo from Thailand – after 15 years</a>
+    </div>
+
+  </section>
+
+  <!--Inspiration-->
+
+  <h1 class="section">INSPIRATION</h1>
+  <section class="inspiration">
+
+    <div class="inspiration-box-mangosorbet">
+      <a class="inspiration-box-image"> <img
+          src="https://assets.icanet.se/e_sharpen:80,q_auto,dpr_1.25,w_718,h_718,c_lfill/imagevaultfiles/id_207516/cf_259/mangosorbet_med_kokos.jpg"
+          alt="mangosorbet"></a>
+      <a class="inspiration-box-headline" href="https://www.ica.se/recept/mangosorbet-med-kokos-725888/o">Mangosorbet
+        med
+        kokos</a>
+    </div>
+
+    <div class="inspiration-box-nachos">
+      <a class="inspiration-box-image"> <img
+          src="https://assets.icanet.se/e_sharpen:80,q_auto,dpr_1.25,w_718,h_718,c_lfill/imagevaultfiles/id_208261/cf_259/gratinerade_nachos_med_mangosalsa.jpg"
+          alt="nachos"></a>
+      <a class="inspiration-box-headline"
+        href="https://www.ica.se/recept/gratinerade-nachos-med-mangosalsa-725925/">Gratinerade nachos med mangosalsa</a>
+    </div>
+
+    <div class="inspiration-box-våfflor">
```

Comment:
Class names should always be in English (all code should always be in English actually)